pkg/controllers/mpi_job_controller.go

```diff
@@ -860,11 +862,20 @@ func newLauncherRoleBinding(mpiJob *kubeflow.MPIJob) *rbacv1.RoleBinding {
 // discover the MPIJob resource that 'owns' it.
 func newWorker(mpiJob *kubeflow.MPIJob, desiredReplicas int32, gpus int) *appsv1.StatefulSet {
 	labels := map[string]string{
-		"app": mpiJob.Name + workerSuffix,
+		// "app": mpiJob.Name + workerSuffix,
```

Comment:
If we do not need the field, we could just remove it.

Reply:
Done.